#! /usr/bin/env python

# Wheel slip plugin configuration

PACKAGE='gazebo_plugins'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#       Name                                      Type      Reconf level   Description                                                                  Default    Min   Max
gen.add("slip_compliance_unitless_lateral",       double_t, 0,             "Unitless slip compliance (slip / friction) in the lateral direction.",      -1.0,      -1.0,  20.0)
gen.add("slip_compliance_unitless_longitudinal",  double_t, 0,             "Unitless slip compliance (slip / friction) in the longitudinal direction.", -1.0,      -1.0,  20.0)

exit(gen.generate(PACKAGE, "wheel_slip",     "WheelSlip"))
